x-pack/plugins/xpack_main/server/lib/register_oss_features.ts

```diff
@@ -143,6 +143,13 @@ const kibanaFeatures: Feature[] = [
         },
         ui: [],
```

Comment:
I might be able to throw a PR together fairly quickly to hide the Advanced Settings save controls, if you want to add a `showWriteControls` capability, or similar. Up to you!

Reply:
> LGTM! Optional comment, otherwise feel free to merge

That would be super awesome! You mind if we do so in a separate PR though? I'd like to merge this so we can demo it for the product review.

Reply:
Good with me!